src/main/java/duke/task/StringManipulator.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+public class StringManipulator {
+    
+    public StringManipulator(){
+        // TODO: 02-Feb-21
+    }
+
+    public static String getFirstWord(String input){
+        return input.split(" ")[0];
+    }
+
+    public static String getStringAfterSlash(String input){
+        return input.substring(input.indexOf("/")+1+2+1);
+    }
+
+    public static String getIndexOfStringAfterWhiteSpace(String input){
+        int firstWhiteSpaceIndex = input.indexOf(" ");
+        if (firstWhiteSpaceIndex == -1){
+            return input.substring(input.length());
+        }
+        return input.substring(firstWhiteSpaceIndex+1);
+    }
+
+    public static String getStringAfterWhiteSpaceAndBeforeSlash(String input){
+        int firstWhiteSpaceIndex = input.indexOf(" ");
+        int firstSlashIndex = input.indexOf("/");
+        if (firstWhiteSpaceIndex == -1) {
+            return input.substring(input.length());
+        }
+        if (firstSlashIndex == -1) {
+            return input.substring(firstWhiteSpaceIndex+1, input.length());
+        }
+        return input.substring(firstWhiteSpaceIndex+1, firstSlashIndex);
+    }
+
+    public static int getTaskNumberDone(String input){
+        return Integer.parseInt(input.substring(5));
+    }
+
+    public static int getTaskNumberDeleted(String input){
+        return Integer.parseInt(input.substring(7));
```

Comment:
Can consider declaring these magic numbers as constants.

Reply:
Thanks, I have made the changes :)